packages/http-client/src/http-client.spec.ts

```diff
@@ -207,6 +208,46 @@ describe('generic http client', () => {
       jest.restoreAllMocks();
     });
 
+    function middleWareBuilder(appendedText: string): Middleware<HttpResponse> {
```

Comment:
```suggestion
    function middlewareBuilder(appendedText: string): Middleware<HttpResponse> {
```

[req] middleware is closed compound word and should therefore not have a capital "W"